ct/alpine.sh

```diff
@@ -24,7 +24,7 @@ var_disk="0.1"
 var_cpu="1"
 var_ram="512"
 var_os="alpine"
-var_version="3.19"
+var_version="3.20"
```

Comment:
Just a question, does this always install the latest minor version eg. 3.20.3 ?

Reply:
Yep ^^ that what proxmox outputs with pveam. They are very up to date